servicetalk-concurrent-api/src/main/java/io/servicetalk/concurrent/api/PublisherAsBlockingIterable.java

```diff
@@ -54,7 +61,7 @@
     private final int queueCapacityHint;
 
     PublisherAsBlockingIterable(final Publisher<T> original) {
-        this(original, 16);
+        this(original, 8);
```

Comment:
current min forced by JCTools is 16 anyways, but will consider followup contribution to allow for smaller initial queue size.

[1] https://github.com/JCTools/JCTools/blob/master/jctools-core/src/main/java/org/jctools/queues/SpscUnboundedArrayQueue.java#L32